std_msgs/String roomName
geometry_msgs/Point[] corners